metrics/web/test/common/presentation/state/projects_notifier_test.dart

```diff
@@ -0,0 +1,179 @@
+import 'dart:async';
+
+import 'package:metrics/common/domain/entities/persistent_store_error_code.dart';
+import 'package:metrics/common/domain/entities/persistent_store_exception.dart';
+import 'package:metrics/common/domain/usecases/receive_project_updates.dart';
+import 'package:metrics/common/presentation/models/persistent_store_error_message.dart';
+import 'package:metrics/common/presentation/state/projects_notifier.dart';
+import 'package:metrics_core/metrics_core.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import '../../../test_utils/matcher_util.dart';
+
+void main() {
+  group("ProjectsNotifier", () {
+    final receiveProjectUpdatesMock = ReceiveProjectUpdatesMock();
+    ProjectsNotifier projectsNotifier;
+    const projects = [
+      Project(id: 'id1', name: 'name1'),
+      Project(id: 'id2', name: 'name2'),
+    ];
```

Comment:
Should we add the tests to verify that Projects are converted to ProjectModels correctly and projectModels updated?